docs/pages/blog/mui-x-end-v6-features.md

```diff
@@ -0,0 +1,190 @@
+  </a>
+</div>
+
+We are happy to share the newest MUI X features, marking the release of [MUI X v6.18.0](https://github.com/mui/mui-x/releases/tag/v6.18.0) and sealing v6 with new components and solid functionalities.
+
+As we gear up to develop the next major version, let's review the notable new features introduced since our last [blog post](https://mui.com/blog/mui-x-mid-v6-features/).
+
+## Table of contents
+
+- [New stable components 🎉](#new-stable-components)
+  - [Charts](#charts)
+  - [Tree view](#tree-view)
+- [Date Pickers](#date-pickers)
+  - [Smoothened animations](#smoothened-animations)
+  - [Clearable Field](#clearable-field)
+  - [Customization Playgrounds](#customization-playgrounds)
+- [Data Grid](#data-grid)
+  - [Column Autosizing](#column-autosizing) [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+  - [New examples](#new-examples)
+  - [Sparkline as a column type](#sparkline-as-a-column-type)
+  - [Performance improvements](#performance-improvements)
+- [What's next](#whats-next)
+- [Feedback](#feedback)
+
+## New stable components
+
+### Charts
+
+The stable version of MUI X charts supports the most commonly used chart plots you'll need in your day-to-day applications.
+
+Featuring lines, areas, bars, pie charts and scatter plots, the latest versions also include:
+
+- **Animations**
+
+  Animations on bars, pie charts, and more to come.
+
+  <video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop playsinline width="680" height="306">
+    <source src="/static/blog/mui-x-end-v6-features/bar-chart-animations.mp4" type="video/mp4">
+  </video>
+
+- **Horizontal Bar Chart Support**
+
+  Recently added, the horizontal bars can broaden your data visualization spectrum, especially when you need to display long labels for multiple categories.
+  Check out Bar's [layout documentation](https://mui.com/x/react-charts/bars/#layout) for more details.
+
+- **Improved Text Customization**
+
+  Customize the legend text to your heart's content, making your charts more readable and visually
+  appealing.
+
+Get started with charts [now!](https://mui.com/x/react-charts/#getting-started)
+
+### Tree view
+
+After an extensive period of development and refinement in the lab and pre-release packages, and with the invaluable feedback we've received from you, the community, we're happy to share that the package is now officially stable!
+
+<video preload="metadata" style="margin-bottom: 10px;" autoplay muted loop playsinline width="680" height="400">
+  <source src="/static/blog/mui-x-end-v6-features/tree-view-gmail.mp4" type="video/mp4">
+</video>
+
+Get started with tree view [now!](https://mui.com/x/react-tree-view/getting-started/)
+
+## Date Pickers
+
+### Smoothened animations
+
+The animations have been smoothened, especially on Android devices, ensuring a more fluid user experience.
```

Comment:
Are you referring to https://github.com/mui/mui-x/pull/9864? 🤔 
In this case, it's probably not the most correct formulation, but it won't hurt. 👌 
In short, we simply fixed an issue where animations were reduced by default on all Android devices.